Cakes-Co/index.html

```diff
@@ -33,7 +33,7 @@
       <section class="p-sec">
         <h1>Welcome</h1>
         <p>
-          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quae
+          Lorem6 ipsum dolor sit amet consectetur, adipisicing elit. Quae
```

Comment:
Looks like you have a small accidental change here 😄 
Whenever you commit and push changes, it's a good idea to view the "diff" in VSCode. This can help us avoid small accidental changes like this.